installation/resources/installer-local.yaml

```
@@ -194,19 +195,79 @@ spec:
               namespace: kyma-installer
             type: Opaque
             EOF
-
             timeout 60s bash -c 'until kubectl create -f /tmp/secret.yaml ; do sleep 2; done'
-
         volumeMounts:
           - mountPath: /etc/certs
             name: helm-certs
             readOnly: true
+      - name: 2to3
+        image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20200529-34b39bd3
+        terminationMessagePolicy: "FallbackToLogsOnError"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -e
+          HELM_2_BINARY=$(which helm)
+          HELM_3_BINARY=$(which helm3)
+          SECRET_NAME="helm-secret"
+          NAMESPACE="kyma-installer"
+          CHART_BLACKLIST=( "gateway-0.0.1" )
```

Could you also add the `application-0.0.1` chart to the blacklist?
```suggestion
          CHART_BLACKLIST=( "gateway-0.0.1" "application-0.0.1" )
```